CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ Release report: TBD
 
 ### Changed
 
+- Skip `test_authd_ssl_options` cases that use TLS 1.1 causing errors on several OpenSSL versions \- (Tests)
```

Comment:
The link is missing

Reply:
Fixed on commit 49b52bacabc8db4ba150298075ff0a2bfdb876a1